### DISCUSSION TITLE
1 Jingle 1 Song

### DISCUSSION THREAD

**Discussion by radiobellissima** [AzuraCast/AzuraCast#6212] [Q&A]
I would like to have a solution to make 1 jingle and 1 song play. Any software can handle this only with azuracast is impossible to do. First it counts jingles as songs and then skips them and then plays 2 or 3 consecutively. It would be enough to mark the jingles as such and eliminate them from the track count and at the same time understand the reason why, at a certain point, it plays many of them in a row. I hope for a solution.

**Comment by Mozitux** [comment]
Hi, just create a playlist for your jingles, then set that playlist to Jingle mode, with shuffle and playlist type to 'Once every x tracks'.
At the bottom of the tab, set the 'Number of songs between each play' value to 1.

![B-01](https://user-images.githubusercontent.com/19860756/232610286-f9ad79b3-4805-4377-9137-d0e4815b5966.PNG)

**Comment by radiobellissima** [comment]
ok, try it and then let me know if it works :-)
